### DISCUSSION TITLE
deployer.org is down

### DISCUSSION THREAD

**Discussion by sglazov** [deployphp/deployer#3824] [General]
I'm using https://github.com/deployphp/action, 15 minutes ago on release I got an error that deployer.org is unavailable. Checked separately - and it's not pinging.

I don't understand yet, is it my local problem _(network/provider/blocking)_ or a global problem with deployer?

<details>

<summary>ping and curl errors</summary>

```bash
$ curl -L https://deployer.org/manifest.json
...
curl: (28) Failed to connect to deployer.org port 443 after 134183 ms: Connection timed out
Error:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
```

```bash
PING deployer.org (172.105.69.92): 56 data bytes
Request timeout for icmp_seq 0
Request timeout for icmp_seq 1
Request timeout for icmp_seq 2
Request timeout for icmp_seq 3
Request timeout for icmp_seq 4
Request timeout for icmp_seq 5
```
</details>

**Comment by LucaGallinari** [comment]
We got this error too both locally and remotely (github action)

**Comment by sandro-magenio** [comment]
I'm also having trouble downloading the deployer and running my pipelines, as all of them are failing. 
Additionally, the website is unreachable.

**Comment by antonmedv** [comment]
Yeap. It is down.

**Reply by antonmedv** [reply]
Trying to bring deployer.org up.

**Reply by antonmedv** [reply]
Apparently I will need help from professional sysops. Does anybody know how to configure Ubuntu 22.04 to work with two IP4 addresses? (╯°□°)╯︵ ┻━┻

**Reply by vitalykarasik** [reply]
BTW, if deployer.org hosts just static files, we can migrate it to AWS S3.

**Reply by sandro-magenio** [reply]
Hi Anton, 
I think that on Ubuntu 22.04, you need to modify the file: /etc/netplan/50-cloud-init.yaml.

**Reply by antonmedv** [reply]
No such file /etc/netplan/50-cloud-init.yaml on my server

**Comment by antonmedv** [comment]
deployer.org is up.

**Reply by LucaGallinari** [reply]
Thank you!

**Reply by lamasfoker** [reply]
Thanks!

**Comment by sglazov** [comment]
Thanks!
